fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -2059,6 +2059,12 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true)
     public static long max_automatic_partition_number = 4096;
 
+    /**
+     * Used to limit num of partition for load open partition number
+     */
+    @ConfField(mutable = true)
+    public static long max_load_open_partition_number = 32;
```

Comment:
What's the usage of this parameter, and is it different with `max_automatic_partition_number` (above)?